app/main.py

```diff
@@ -34,6 +37,7 @@ def wash_single_car(self, car: Car) -> None:
 
     def rate_service(self, rate: int) -> None:
         self.average_rating = round((self.average_rating
-                                     * self.count_of_ratings + rate)
+                                     * self.count_of_ratings
```

Comment:
same here. formatting is according to best practices, but it is better to decompose such huge formula